packages/block-library/src/navigation-submenu/index.php

```diff
@@ -239,6 +239,10 @@ function render_block_core_navigation_submenu( $attributes, $content, $block ) {
 			$inner_blocks_html .= $inner_block->render();
 		}
 
+		if ( strpos( $inner_blocks_html, 'current-menu-item' ) ) {
```

Comment:
So bad we need to string match.

[This kind of check](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation-link/index.php#L153) would be nicer, but I guess it's not possible :)

Reply:
Yeah, that would indeed be better... But unfortunately, I don't think it's possible to detect the current menu item using a check like that since the class is getting added dynamically and is not an attribute or something that is saved 🤷 😞